dcr/azure-pipelines.yml

```diff
@@ -0,0 +1,104 @@
+parameters:
+  - name: scenarios
+    type: object
+    default:
```

Comment:
when does these defaults take place?

Reply:
Defaults are included whenever the pipeline is triggered automatically (daily runs, PR runs, etc). 
When you run it manually, you get a UI which shows all these parameters where you can manually add/remove them. Something like this - 

![image](https://user-images.githubusercontent.com/41490930/138092667-bf4eed0e-3f97-445c-b720-8b430fcbc60d.png)